pkg/storage/test/tuples.go

```diff
@@ -352,11 +320,11 @@ func ReadChangesTest(t *testing.T, datastore storage.OpenFGADatastore, tokenSeri
 		seenObjects := map[string]struct{}{}
 
 		var changes []*openfgav1.TupleChange
-		var continuationToken []byte
+		var continuationToken string
```

Comment:
Suggestion: rename `continuationToken` -> `ulidOfTuple` (or whatever name you deem fit)